juniper_official/interface/interface-error-details.yml

```diff
@@ -1,11 +1,14 @@
 ---
 intfaceStats:
   rpc: get-interface-information
+  command: /usr/sbin/cli -c 'show interfaces extensive | display xml | save /tmp/interface-ext.txt'
```

Comment:
Change file name to /tmp/insights-interface-ext.xml

Reply:
Updated file name.